Library/Formula/pmd.rb

```diff
@@ -2,23 +2,26 @@
 
 class Pmd < Formula
   homepage "http://pmd.sourceforge.net/"
-  url "https://downloads.sourceforge.net/project/pmd/pmd/5.1.3/pmd-src-5.1.3.zip"
-  sha1 "da6023d73e078444a7b76e8a7a0cfbeb421221e4"
+  url "https://downloads.sourceforge.net/project/pmd/pmd/5.3.2/pmd-src-5.3.2.zip"
+  sha1 "036cdb7583de3ea0d24e7486bf7b9f9f14bfb1ec"
+
+  depends_on "maven" => :build
+
+  patch :DATA
```

Comment:
Why is this patch required? If it’s not Homebrew-specific, could you report it upstream and add a link to the upstream issue in a comment? In any case, could you add a comment to explain why is it necessary? Thanks!